include/mlir/Dialect/SPIRV/SPIRVBase.td

```diff
@@ -143,14 +145,18 @@ def SPV_AnyPtr : Type<SPV_IsPtrType, "any SPIR-V pointer type">;
 def SPV_AnyArray : Type<SPV_IsArrayType, "any SPIR-V array type">;
 def SPV_AnyRTArray : Type<SPV_IsRTArrayType, "any SPIR-V runtime array type">;
 def SPV_AnyStruct : Type<SPV_IsStructType, "any SPIR-V struct type">;
+// TODO(denis0x0D): Find the better way to define an index type.
```

Comment:
We should not introduce this type here as explained earlier.

Reply:
Fixed.